src/bitmessagemain.py

```diff
@@ -25,9 +25,10 @@
 import errno
 import socket
 import ctypes
+import subprocess
```

Comment:
imports should follow the order.

import  should be first.
from statements should be next to them.

Reply:
This PR is not about style, I have #1189 for that. Here I retained the original order of imports.

Reply:
And also, can you give me a proof link? I don't see such requirements in pep8.

Reply:
https://www.python.org/dev/peps/pep-0008/#imports has a requirement regarding grouping, and while alphabetic order isn't a part of PEP8 as far as I see, it's a commonly used standard.